ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -543,7 +543,7 @@ export default class ConfirmTransactionBase extends Component {
         toName={toName}
         toAddress={toAddress}
         showEdit={onEdit && !isTxReprice}
-        action={actionKey && this.context.t(actionKey) || getMethodName(name) || this.context.t('contractInteraction')}
+        action={this.context.t(actionKey) || getMethodName(name) || this.context.t('contractInteraction')}
```

Comment:
```suggestion
        action={actionKey || getMethodName(name) || this.context.t('contractInteraction')}
```

Reply:
does this remove the [undefined] ?

Reply:
It does, but will need to make a separate PR that targets `develop`

Reply:
It will fallback to `contractInteraction`.